src/ORM/Query.php

```diff
@@ -1132,8 +1132,10 @@ protected function _execute(): ResultSetInterface
         }
 
         $statement = $this->getEagerLoader()->loadExternal($this, $this->execute());
+        $resultset = new ResultSet($this, $statement->fetchAll('assoc') ?: []);
+        $statement->closeCursor();
```

Comment:
Is fetchAll() not closing the cursor properly?

Reply:
I probably just copied it from somewhere and might not be actually needed.